src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  &__title {
+    padding-inline: 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    position: absolute;
+    top: 206px;
+    transition-duration: $transition;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__product-code {
+    padding-inline: 16px;
+    position: absolute;
+    top: 246px;
+    color: #616070;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__price {
+    padding-inline: 16px;
+    width: 166px;
+    position: absolute;
+    top: 316px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__price-price {
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    padding-inline: 16px;
+    display: flex;
+  }
+
+  &__buy {
+    color: #fff;
+    font-family: Roboto, sans-serif;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    transition-duration: $transition;
+
+    position: absolute;
+    top: 350px;
```

Comment:
Using absolute positioning for the buy button might cause issues with responsiveness. Consider using relative positioning or a layout model like flexbox or grid to ensure the button adjusts well on different screen sizes.